### ISSUE TITLE
Question about Config.disableNPC

### ISSUE THREAD

**Issue by Legacy-TacticalGamingInteractive** [TanguyOrtegat/jb_vehiclelock#4]
Config.disableNPC = true

Is there a way to make this only 50% of the time?
So... sometimes the car is unlocked. This way people can only steal the cars from NPC occasionally.

**Comment by TanguyOrtegat** [comment]
yes ofc you can, just write an additional if statement above this line: https://github.com/TanguyOrtegat/jb_vehiclelock/blob/master/jb_vehiclelockclient.lua#L189
if math.random(0,100) < 50 then
...

something like that

**Comment by Legacy-TacticalGamingInteractive** [comment]
Thank you I will try this!

Did you ever get a chance to add animation for lock/unlock ?
 Or maybe... pNotify saying it is locked/unlocked?

**Comment by TanguyOrtegat** [comment]
no annimation for the moment ( i don't update it anymore) and notify is already saying locked/unlocked normally
